crowbar_framework/app/models/backup.rb

```diff
@@ -159,14 +159,19 @@ def save_archive
       return false
     end
 
+    if data.join("meta.yml").exist?
+      meta = YAML.load_file(data.join("meta.yml"))
```

Comment:
That requires a begin and rescue block with a catch for the correct error class.